src/components/Calculator.js

```diff
@@ -1,108 +1,202 @@
 /* eslint-disable jsx-a11y/click-events-have-key-events */
 /* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
 /* eslint-disable react/prefer-stateless-function */
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import '../App.css';
 import calculate from '../logic/calculate';
 
-class Calculator extends Component {
-  constructor(props) {
-    super(props);
+// class Calculator extends Component {
+//   constructor(props) {
+//     super(props);
 
-    this.state = {
-      total: 0,
-      next: '',
-      operation: '',
-    };
-  }
+//     this.state = {
+//       total: 0,
+//       next: '',
+//       operation: '',
+//     };
+//   }
 
-  eventHandler = (e) => {
+//   eventHandler = (e) => {
+//     const val = e.target.textContent;
+//     this.setState((obj) => calculate(obj, val));
+//   };
+
+//   render() {
+//     const { total, next, operation } = this.state;
+//     const results = `
+//     ${
+//   total || operation || next
```

Comment:
- [ ] Kindly remove all unused commented codes in order for your codebase to be clean and neat. For this project and every other project, you are expected to have only a working and meaningful code to the program and a comment that explains a feature or section of your code but not unused commented code. Please remove all commented codes in your codebase in order for your project to be complete👍